.github/workflows/publish-as-package.yaml

```diff
@@ -0,0 +1,38 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
```

Comment:
This is a basic github workflow for easy publishing when you release in github. It does everything automatically, you'll find more details under the url. For it to work you need to add your `PYPI_API_TOKEN` in your projects secrets (see last line of the file)

Reply:
I've checked the link and I think I understood a little bit. But it is still cloudy (I'll have to get used to it). Still, did not get if why in line 35 it appears `pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29` instead of something like `pypa/gh-action-pypi-publish@release/v1`. Also, from what I've checked, it would be better to use TestPyPi first, so I'll try to change a little bit after accepting the PR (will try to follow some of [this](https://github.com/pypa/gh-action-pypi-publish#usage)). Since I created the `TEST_PYPI_API_TOKEN` instead of `PYPI_API_TOKEN` I believe nothing will happen for now.

Reply:
Also, my idea behind the review process was adding changes in the `joss` branch, not in `master`. Eventually, after it, I would merge the two branches.
But, since you forked master and did the PR on it, I'll have to think more on how to integrate the previous changes in joss with  master later :sweat_smile: